modules/globebrowsing/scripts/layer_support.lua

```diff
@@ -8,6 +8,55 @@ openspace.globebrowsing.documentation = {
         Name = "createHeightLayers",
         Arguments = "table",
         Documentation = "Creates a table used in the 'HeightLayers' of a RenderableGlobe."
+    },
+    {
+        Name = "createTemporalGibsGdalXml",
```

Comment:
It's okay to have them as arguments.. Lua allows to pack and unpack tables into arguments, so it doesn't really matter from a usage